iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/posix_call.inl

```diff
@@ -163,8 +163,8 @@ PosixCallEvaluator<ReturnType>::evaluate() const&& noexcept
     }
     else if (!m_details.hasSilentErrno)
     {
-        std::cerr << m_details.file << ":" << m_details.line << " { " << m_details.callingFunction << " -> "
-                  << m_details.posixFunctionName << " }  :::  [ " << m_details.result.errnum << " ]  "
+        std::cerr << m_details.file << ":" << std::dec << m_details.line << " { " << m_details.callingFunction << " -> "
+                  << m_details.posixFunctionName << " }  :::  [ " << std::dec << m_details.result.errnum << " ]  "
```

Comment:
`std::dec` is persistent. Please restore the flags after usage, e.g.
```cpp
auto flags = std::cerr.flags();
std::cerr << std::hex << 0xAFFE << std::endl;
std::cerr.setf(flags);
```
Ideally where it was changed from `std::dec`

Reply:
@elBoberido you are completely right and `std::dec` is the default but sadly somewhere else someone (maybe me) did not restore the flags and therefore I had a hex output here.

Restoring here the flags would not help since then we would have again the default hex output :/ But I can sniffle around and try to find this hex output and fix it.